standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/DatabaseProduct.java

```diff
@@ -749,6 +753,22 @@ public Object getBoolean(boolean val) {
     return val;
   }
 
+  public Object convertDateValue(Object dateValue) {
```

Comment:
that is a weird signature for a method that is supposed to accept a String and return String.
also not clear what kind of conversion it's doing

Reply:
1. why don't we rename it to `normalizeDate` or something
2. why input and output types are Object and not String
3. why this util method is in DatabaseProduct class (not MetaStoreUtils), has nothing to do with it